lib/browser/rpc-server.ts

```diff
@@ -71,6 +71,18 @@ if (BUILDFLAG(ENABLE_DESKTOP_CAPTURER)) {
 
     return typeUtils.serialize(await desktopCapturer.getSourcesImpl(event, options));
   });
+
+  ipcMainInternal.handle(IPC_MESSAGES.DESKTOP_CAPTURER_SET_SKIP_CURSOR, function (event, sourceId, skipCursor, stack) {
+    logStack(event.sender, 'desktopCapturer.setSkipCursor()', stack);
+    const customEvent = emitCustomEvent(event.sender, 'desktop-capturer-set-skip-cursor');
```

Comment:
Can you add documentation for this event?

Reply:
Sure. Added the documentation for the event. Added basic tests for the api.